src/ck3/tables/misc.rs

```diff
@@ -25,23 +25,23 @@ pub const ARTIFACT_HISTORY: &[&str] = &[
     "reforged",
 ];
 
-// LAST UPDATED CK3 VERSION 1.10.0
-// TODO: parse it from dlc_metadata/ ? Unfortunately Tours and Tournaments
-// is an exception.
+// LAST UPDATED CK3 VERSION 1.11.3
+// TODO: parse it from dlc_metadata/
 pub const DLC_CK3: &[&str] = &[
+    "Garments of the Holy Roman Empire",
     "Fashion of the Abbasid Court",
     "The Northern Lords",
-    "Garments of the Holy Roman Empire",
-    "The Fate of Iberia",
     "The Royal Court",
+    "The Fate of Iberia",
     "Friends and Foes",
-    "tours_and_tournaments",
+    "Tours and Tournaments",
```

Comment:
Are you sure about this change? I still see `has_dlc = tours_and_tournaments` in the vanilla script